common/test/acceptance/tests/studio/test_studio_course_team.py

```diff
@@ -174,6 +187,25 @@ def test_admins_can_delete_other_users(self):
         self.log_in(self.other_user)
         self._assert_current_course(visible=False)
 
+    def test_admins_can_delete_other_inactive_users(self):
```

Comment:
To cater a scenario in which front end is involved and a complete flow where user is added in course and is deleted after user's status is inactive. Also all other related scenarios are handled in this fixture in which this test is added.